resources/views/community/components/user/profile-meta.blade.php

```diff
@@ -0,0 +1,68 @@
+@props([
+    'averageCompletionPercentage' => '0.0',
+    'averageFinishedGames' => 0,
+    'averagePointsPerWeek' => 0,
+    'developerStats' => [],
+    'hardcoreRankMeta' => [],
+    'recentPointsEarned' => [],
+    'socialStats' => [],
+    'softcoreRankMeta' => [],
+    'totalHardcoreAchievements' => 0,
+    'totalSoftcoreAchievements' => 0,
+    'userClaims' => null, // ?array
+    'userMassData' => [],
+    'username' => '',
+])
+
+<?php
+use App\Site\Enums\Permissions;
+
+$jrDevPermission = Permissions::JuniorDeveloper;
+?>
+
+<div class="relative">
+    <x-user.profile.primary-meta
+        :hardcoreRankMeta="$hardcoreRankMeta"
+        :softcoreRankMeta="$softcoreRankMeta"
+        :userMassData="$userMassData"
+        :username="$username"
+    />
+
+    <div class="mb-2 sm:ml-[142px] md:ml-0 md:absolute md:top-0.5 md:right-0 lg:relative lg:ml-[142px] xl:absolute xl:ml-0">
+        <x-user.profile.social-interactivity :username="$username" />
+    </div>
+</div>
+
+<hr class="border-embed-highlight mb-2" />
```

Comment:
This `hr` is nearly imperceptable.

![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/0d783b95-84a5-4ee8-8cac-b7cce0847a60)

Contrast greatly increased in highlighted region.

Reply:
It is tough to see on dark and black and easy to see on light mode. I'm indifferent on whether or not it should stay; I don't think it's hurting anything.

Reply:
It's definitely not hurting anything. It's just one of those things that I didn't see for a while, so when it did catch my eye I wondered if it was a bug.

Reply:
Unless you're opposed, I'd like to keep this for now. On one of my monitors, I can't see the `<hr />` while in dark mode. On my work monitor (higher quality / color accuracy), it subtly adds some visual distinction between the two sections that my eye picks up on.